src/test/java/org/openrewrite/java/migrate/javax/AddTransientAnnotationToPrivateAccessorTest.java

```diff
@@ -0,0 +1,462 @@
+    }
+
+    @DocumentExample
+    @Test
+    void addTransientToMethodReturningIdentifier() {
+        //language=java
+        rewriteRun(
+          java(
+            """
+              package entities;
+
+              import javax.persistence.Entity;
+              import javax.persistence.Id;
+
+              @Entity
+              public class PrivateAccessor  {
+                  private int id;
+                  private int nonPersistentField;
+
+                  @Id
+                  public int getId() {
+                      return id;
+                  }
+
+                  private int getNonPersistentField() {
+                      return nonPersistentField;
+                  }
+              }
+              """,
+            """
+              package entities;
+
+              import javax.persistence.Entity;
+              import javax.persistence.Id;
+              import javax.persistence.Transient;
+
+              @Entity
+              public class PrivateAccessor  {
+                  private int id;
+                  private int nonPersistentField;
+
+                  @Id
+                  public int getId() {
+                      return id;
+                  }
+
+                  @Transient
+                  private int getNonPersistentField() {
+                      return nonPersistentField;
+                  }
+              }
+              """
+          )
+        );
+    }
+
+    @Test
+    void addTransientToMethodReturningFieldAccess() {
+        //language=java
+        rewriteRun(
+          java(
+            """
+              package entities;
+
+              import javax.persistence.Entity;
+              import javax.persistence.Id;
+
+              @Entity
+              public class PrivateAccessor  {
+                  private int id;
+                  private int nonPersistentField;
+
+                  @Id
+                  public int getId() {
+                      return id;
+                  }
+
+                  private int getNonPersistentField() {
+                      return this.nonPersistentField;
+                  }
+              }
+              """,
+            """
+              package entities;
+
+              import javax.persistence.Entity;
+              import javax.persistence.Id;
+              import javax.persistence.Transient;
+
+              @Entity
+              public class PrivateAccessor  {
+                  private int id;
+                  private int nonPersistentField;
+
+                  @Id
+                  public int getId() {
+                      return id;
+                  }
+
+                  @Transient
+                  private int getNonPersistentField() {
+                      return this.nonPersistentField;
+                  }
+              }
+              """
+          )
+        );
+    }
+
+    @Test
+    void doNotChangePublicOrProtectedGetter() {
```

Comment:
The spec and tests don't seem to mention `package private` cases. Would we want to make explicit what we do with those in the tests? Right now it seems we only change `private` cases.

Reply:
I think since the spec doesn't mention it, it may be best to only stick with the explicit `private` case.